client/src/components/header/index.js

```diff
@@ -0,0 +1,115 @@
+import React, { useContext } from 'react';
+import { Link, NavLink, withRouter } from 'react-router-dom';
+import './style.css';
+import { Modal, Menu, Dropdown } from 'antd';
+import { FaBookmark, FaUserAlt, AiFillHome, IoMdLogOut } from 'react-icons/all';
+import Logo from '../../assets/Logo.svg';
+import { AuthContext } from '../../container/authContext';
+
+const Header = ({ history }) => {
+  const { userInfo, isAuth, showLoginModal, signOut } = useContext(AuthContext);
+  // console.log(userInfo);
+
+  const info = () => {
+    Modal.info({
+      title: 'Course Search',
+      content: (
+        <div>
+          <p>
+            {' '}
+            The place where you will find the best courses on the internet. With
+            our app we want to help the people through searching among different
+            platform about their favorite courses, by one click you will find
+            all the top courses.
+          </p>
+        </div>
+      ),
+      onOk() {},
+    });
+  };
+  const menu = (
+    <Menu>
+      <Menu.Item key="1" onClick={() => history.push('/')}>
+        <AiFillHome />
+        Home
+      </Menu.Item>
+
+      {isAuth && (
+        <Menu.Item key="2" onClick={() => history.push('/FavoritePage')}>
```

Comment:
https://github.com/GSG-G8/courses-search/blob/ba373744a308797aab2e660a54a3d81ed10358d5/client/src/constants/router.js#L2
based on this change the path to ```/favoriteCourses```